src/main/java/uk/gov/companieshouse/api/accounts/CompanyAccountsApplication.java

```diff
@@ -58,6 +62,8 @@ public ObjectMapper objectMapper() {
 
     @Override
     public void addInterceptors(final InterceptorRegistry registry) {
+        registry.addInterceptor(loggingInterceptor)
+            .addPathPatterns("/**").excludePathPatterns("/healthcheck");
```

Comment:
You don't need `addPathPatterns("/**")` here; if not provided all paths are assumed included. Just exclude the healthcheck endpoint.